src/linear_algebra/finsupp_vector_space.lean

```diff
@@ -80,28 +80,48 @@ end dim
 end finsupp
 
 section vector_space
-universes u v
-variables {α : Type u} {β γ : Type v}
+universe variables u v w
```

Comment:
```suggestion
universes u v w
```
Or is this doing something I don't understand?

Reply:
`universes` did not work here and @fpvandoorn suggested to use `universe variables` instead.
The issue with `universes` is that the universes introduced by this keyword do not get replaced by metavariables once a lemma has been proven. So if you prove a lemma using universe `u`, you can only apply it to universe `u` in other lemmas of the same section.

Reply:
Oh, that's crazy. Maybe we should always use `universe variables` then? Thanks for explaining! (Given that this is not standard practice, maybe include this short explanation above this line?)

Reply:
> Maybe we should always use universe variables then? 

That's what Floris suggested, too.

>maybe include this short explanation above this line?

ok